scripts/pre_commit_linter.py

```diff
@@ -143,6 +143,12 @@
             'extensions/classifiers/',
             'extensions/objects/',
             'extensions/value_generators/')
+    },
+    {
+        'regexp': r"([^ \t\r\n])[ \t]+$",
+        'message': "There should not be any trailing whitespaces.",
```

Comment:
@seanlip: This message will not help a developer while debugging. I intend to make a few changes to the `_check_bad_pattern_in_file` function to display the error message with a line number.

Reply:
Fixed (please see the below comments.)